### PR TITLE
Adds timer displaying time elapsed to active jobs tab.

### PR DESCRIPTION
**Pull request by Kleptobismol**

### CODE REVIEW

File: app/js/components/JobRow.jsx [hunk 1]
```diff
@@ -9,19 +10,23 @@ const JobRow = ({ data, onClick }) => (
             </a>
         </td>
         <td>
-            {data.started}
+            { moment.utc(data.started).format('M-D-Y hh:mm:ss') }
```
Comment (ebolyen) [inline app/js/components/JobRow.jsx]: I don't think these should be `utc`, but I'll verify that.

Comment (ebolyen) [inline app/js/components/JobRow.jsx]: Yeah it is off by 7 hours again. So for these we want just the normal constructor which considers the timezone. 
The date format should also be `YY-MM-DD hh:mm:ss` to match ISO 8601.

File: app/js/reducers/timer.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const initialState = {
```
Comment (ebolyen) [inline app/js/reducers/timer.js]: Remove this file.

File: app/js/components/JobRow.jsx [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-
+import Timer from './Timer'
+var moment = require('moment');
```
Comment (ebolyen) [inline app/js/components/JobRow.jsx]: Change this import to 

```javascript
import moment from 'moment';
```

File: app/js/components/JobRow.jsx [hunk 4]
```diff
@@ -9,19 +10,23 @@ const JobRow = ({ data, onClick }) => (
...
         <td>
-            {data.started}
+            { moment.utc(data.started).format('M-D-Y hh:mm:ss') }
         </td>
         {data.finished ?
             <td>
-            {data.finished}
-            </td> : null
-        }
+            { moment.utc(data.finished).format('M-D-Y hh:mm:ss') }
```
Comment (ebolyen) [inline app/js/components/JobRow.jsx]: Same comment re: utc/formatting

File: app/js/components/Timer.jsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    }
+    render() {
+        const start = this.props.start; 
+        return (
+            <span 
+                ref={(e) => {
+                    this.elem = e;
+                }}
+            >
+                00:00:00
```
Comment (ebolyen) [inline app/js/components/Timer.jsx]: :+1:

File: app/js/components/Timer.jsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+var moment = require('moment');
+
+class Timer extends React.Component {
+
+    componentDidMount() {
+        this.interval = setInterval(() => {
+            const diff = Date.now() - +this.props.start;
+            const formattedDate = moment.utc(diff).format('HH:mm:ss')
+            // var formattedDate = dateFormat(diff, "h:MM:ss");
```
Comment (ebolyen) [inline app/js/components/Timer.jsx]: remove

File: app/js/reducers/timer.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+            ]
+        };
+        return newState;
+    }
+    default:
+        return state;
+    }
+};
+
+export default timerReducer;
```
Comment (ebolyen) [inline app/js/reducers/timer.js]: missing newline at end of file.

File: app/js/components/JobRow.jsx (hunk 1 above)
Comment (Kleptobismol) [inline app/js/components/JobRow.jsx]: Fixed

File: app/js/reducers/timer.js (hunk 2 above)
Comment (Kleptobismol) [inline app/js/reducers/timer.js]: Fixed

File: app/js/components/JobRow.jsx (hunk 3 above)
Comment (Kleptobismol) [inline app/js/components/JobRow.jsx]: Fixed

File: app/js/components/JobRow.jsx (hunk 4 above)
Comment (Kleptobismol) [inline app/js/components/JobRow.jsx]: Fixed

File: app/js/components/Timer.jsx (hunk 6 above)
Comment (Kleptobismol) [inline app/js/components/Timer.jsx]: Fixed

File: app/js/reducers/timer.js (hunk 7 above)
Comment (Kleptobismol) [inline app/js/reducers/timer.js]: Fixed

**Pull request merged**